tests/verifyMongodbSharding_test.js

```diff
@@ -0,0 +1,41 @@
+Feature('MongoDB Sharding tests');
+
+const { adminPage } = inject();
+const pmmFrameworkLoader = `bash ${adminPage.pathToFramework}`;
+const exactMongoDbInfo = new DataTable(['version', 'edition']);
+
+// exactMongoDbInfo.add(['4.0', 'Community']);
+exactMongoDbInfo.add(['4.2', 'Community']);
+exactMongoDbInfo.add(['4.4', 'Community']);
+// exactMongoDbInfo.add(['5.0', 'Community']);
+exactMongoDbInfo.add(['6.0', 'Community']);
+
+Data(exactMongoDbInfo).Scenario(
+  '@PMM-T1539 Verify that MongoDB exporter shows version for mongos instance @mongodb-exporter',
+  async ({
+    I, current,
+  }) => {
+    const { version, edition } = current;
+
+    await I.verifyCommand(`${pmmFrameworkLoader} --mongomagic --with-sharding --pmm2 --mo-version=${version}`);
+    const containerName = await I.verifyCommand(`docker ps --format "table {{.ID}}\\t{{.Image}}\\t{{.Names}}" | grep 'psmdb.*${version}_sharded' | awk -F " " '{print $3}'`);
+    const agentId = await I.verifyCommand(`docker exec ${containerName} pmm-admin list | grep "42002" | awk -F " " '{print $4}'`);
```

Comment:
ideally it is not good to hardcode port, it should be derived from the service Name we add the service with

Reply:
Fixed